File: Facebook.NET/GraphAPI.Array.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class Array<T>
        {
            [DataMember(Name = "count")]
            public int Count { get; set; }

            [DataMember(Name = "items")]
            public T[] Items { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Array.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Array(Of T)
            <DataMember(Name:="count")>
            Public Property Count() As Integer
            <DataMember(Name:="items")>
            Public Property Items() As T()
        End Class
    End Class
End Namespace
